tableau_utilities/test_tableau_utilities.py

```diff
@@ -49,7 +49,27 @@ def test_cli_server_info_args():
     assert args.server == 'us-east-1'
 
 
-# TEST 3: Datasource().DatasourceItems
+# TEST 3: Column()
```

Comment:
Cool.  I wonder if there is a better way to dynamically number these tests or something. hmmmmm.  

Fix looks good to me though!

Reply:
Yeah, maybe we can just remove the numbers. Kind of a pain to update them every time 😆

Reply:
Do you remember why we did it?  I think I started that design patter because if a test fails that made it easy to identify which one failed?  Or perhaps I wanted to know total fails?  I can't really remember at this point.

Reply:
tbh I don't know the exact reason, but I think it was for better identifying test failures yeah.
But since the test errors include the name of the function, I think the numbers are probably not necessary.